lldb/source/Host/windows/Host.cpp

```diff
@@ -363,5 +353,6 @@ void Host::SystemLog(Severity severity, llvm::StringRef message) {
     event_type = EVENTLOG_INFORMATION_TYPE;
   }
 
-  ReportEventW(h, event_type, 0, 0, nullptr, 1, 0, &msg_ptr, nullptr);
+  LPCWSTR messages[] = {argsUTF16.data()};
+  ReportEventW(h, event_type, 0, 0, nullptr, 1, 0, messages, nullptr);
```

Comment:
I would consider retaining this as `&argsUTF16.data()` instead of the `LPCWSTR[]`. Or if you want to use the explicit array, consider replacing the `1` with `countof(messages)`.

Reply:
Could not get it to compile with the `&argsUTF16.data()`, because of the following error: ` cannot convert argument 8 from 'wchar_t **' to 'LPCWSTR *'`.

I used a fixed size array and `std::size`.